### ISSUE TITLE
Unable to bump version.

### ISSUE THREAD

**Issue by lukasturcani** [purescript/spago#682]
I have two PureScript libraries, both of which are in ``purescript/package-sets`` and ``purescript/registry``. These are 

https://github.com/lukasturcani/validated-molecule
 https://github.com/lukasturcani/mol-draw

``mol-draw`` depends on ``validated-molecule``.  ``validated-molecule`` is new, so it was never in Bower at any point. Not sure if that's relevant or not. I am able to successfully run ``spago bump-version --no-dry-run <whatever>`` on ``validated-molecule`` and also ``pulp publish``, as a result, ``validate-molecule`` is successfully on ``Pursuit``.

However, for  ``mol-draw``, running ``spago bump-version --no-dry-run <whatever>`` results in the following error

```
Failed to run: `bower info --json purescript-validated-molecule#v1.0.5`
```

spago version: 0.15.2
pulp version: 15.0.0
purs version: 0.13.8

**Comment by ursi** [comment]
You can see my struggle with this (here)[https://functionalprogramming.slack.com/archives/C04NA444H/p1603210605055300]. Hopefully this can be of use to you or others.

**Comment by ursi** [comment]
my proposed solution to this problem is the following:

if `bower info --json purescript-validated-molecule#v1.0.5` fails, we get the repo from the package set and try
```
bower info --json https://github.com/lukasturcani/validated-molecule.git#v1.0.5
```

If that succeeds, the generated `bower.json` file will include
```
"purescript-validated-molecule": "https://github.com/lukasturcani/validated-molecule.git#^v1.0.5"
```
instead of the normal `"package-name": "semver range"`

**Comment by f-f** [comment]
Fixed in #757
